### ISSUE TITLE
`leptos_axum` uses `tokio/full`

### ISSUE THREAD

**Issue by k9withabone** [leptos-rs/leptos#1835]
**Describe the bug**

`leptos_axum` uses `tokio`'s `full` feature by default. Looking at the `leptos_axum` code it only seems to need the `fs` and `sync` features.

**Leptos Dependencies**

```toml
leptos = { version = "0.5.0", features = ["ssr"] }
leptos_axum = { version = "0.5.0" }
leptos_router = { version = "0.5.0", features = ["ssr"] }
```

**To Reproduce**
1. Create a new package: `cargo new leptos_test`
2. Add the above dependencies to the `Cargo.toml`
3. Review `tokio`'s dependencies: `cargo tree -p tokio`
4. See excess dependencies e.g. `tokio-macros`.

**Expected behavior**
A clear and concise description of what you expected to happen

`leptos_axum` should use the minimum amount of `tokio` features necessary to reduce the number of required dependencies and compile times.

**Additional context**

Just started learning leptos and I'm really liking it so far! One of the first things I do when looking at a new crate is review its features and dependencies so I noticed this potential issue. I could definitely be wrong here and there's something I'm not seeing.

**Comment by gbj** [comment]
Thanks for taking the time to look into it! I'm sure there's no good reason here. If you'd like you can make a PR reducing the feature set and, assuming it compiles, everyone will benefit.

**Comment by gbj** [comment]
Closed by #1846.
